src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -42,27 +42,27 @@ const DragDropcontext: FC<dragDropProps> = ({
           ? 'Sucessfully Assigned Task'
           : 'Something went wrong';
```

Comment:
This condition will never be false because any error returned from API will take the execution in the catch block.
Do the needful